bilingual-module/README

```diff
@@ -26,49 +27,60 @@ if_hfst}}
 
-You can use `./autogen.sh` instead of `./configure` you're compiling from
-source. If you installed any prerequisite language packages using a `--prefix`
-to `./configure`, make sure to give the same `--prefix` to `./configure` here.
 
 Testing
 -------------------------------------------------------------------------------
 
 If you are in the source directory after running make, the following
 commands should work:
 
-    $  echo "TODO test sentence 1" | apertium -d . {{languageCode1}}-{{languageCode2}}
-    TODO test translated sentence 1
+```bash
+$ echo "TODO test sentence 1" | apertium -d . {{languageCode1}}-{{languageCode2}}
+TODO test translated sentence 1
 
-    $ echo "TODO test sentence 2" | apertium -d . {{languageCode2}}-{{languageCode1}}
-    TODO test translated sentence 2
+$ echo "TODO test sentence 2" | apertium -d . {{languageCode2}}-{{languageCode1}}
+TODO test translated sentence 2
+```
 
 After installing somewhere in `$PATH`, you should be able to do e.g.
 
-    $  echo "TODO test sentence 1" | apertium {{languageCode1}}-{{languageCode2}}
-    TODO test translated sentence 1
+```bash
+$ echo "TODO test sentence 1" | apertium {{languageCode1}}-{{languageCode2}}
+TODO test translated sentence 1
+```
+
 
 Files and data
 -------------------------------------------------------------------------------
 
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode1}}-{{languageCode2}}.dix`  - Bilingual dictionary
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode1}}-{{languageCode2}}.t1x`  - Chunking rules for translating into {{languageName2}}
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode2}}-{{languageCode1}}.t1x`  - Chunking rules for translating into {{languageName1}}
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode1}}-{{languageCode2}}.t2x`  - Interchunk rules for translating into {{languageName2}}
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode2}}-{{languageCode1}}.t2x`  - Interchunk rules for translating into {{languageName1}}
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode1}}-{{languageCode2}}.t3x`  - Postchunk rules for translating into {{languageName2}}
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode2}}-{{languageCode1}}.t3x`  - Postchunk rules for translating into {{languageName1}}
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode1}}-{{languageCode2}}.lrx`  - Lexical selection rules for translating into {{languageName2}}
-* `apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode2}}-{{languageCode1}}.lrx`  - Lexical selection rules for translating into {{languageName1}}
-* `modes.xml`                     - Translation modes
+* [`apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode1}}-{{languageCode2}}.dix`](apertium-{{languageCode1}}-{{languageCode2}}.{{languageCode1}}-{{languageCode2}}.dix) - Bilingual dictionary
```

Comment:
Ah, does this link to the file in the repo?

Reply:
Indeed